include/xsimd/arch/generic/xsimd_generic_details.hpp

```diff
@@ -77,6 +77,8 @@ namespace xsimd
     template <class T, class A>
     inline batch<T, A> nearbyint(batch<T, A> const& self) noexcept;
     template <class T, class A>
+    inline batch<as_integer_t<T>, A> nearbyint_as_int(const batch<T, A>& x) noexcept;
```

Comment:
@serge-sans-paille @JohanMabille is the naming ok? the closest in meaning is `nearbyint`, but the behaviour is mapped by `round` + `to_int`.